docs/reference/commandline/dockerd.md

```diff
@@ -1154,6 +1154,7 @@ The following daemon options must be configured for each daemon:
 -g, --graph=/var/lib/docker            Root of the Docker runtime
 -p, --pidfile=/var/run/docker.pid      Path to use for daemon PID file
 -H, --host=[]                          Daemon socket(s) to connect to
+--iptables                             Enable addition of iptables rules
```

Comment:
yeah, had that first but it will change in the `--help` output if we switch to Cobra for this. Perhaps we can keep it here; let me update